test/mocha/mig-login.js

```diff
@@ -0,0 +1,27 @@
+/* global describe before after it expect Event */
+import closest from '../../app/utils/closest'
+import {dispatch} from '../../app/utils/events'
+import app from '../..' // eslint-disable-line no-unused-vars
```

Comment:
Fundamental problem is the fact that our web components are being tested as an application instead of atomically. Until there's a solution to the `document.registerElement` problem I suggest begrudgingly tolerating this one-line boilerplate in all tests.